pkg/moov/application_models.go

```diff
@@ -0,0 +1,46 @@
+package moov
+
+import "time"
+
+type ApplicationScope string
```

Comment:
should we eventually have an enum of publicly known scopes here, or just string?

Reply:
Maybe, right now Just going to leave ti a string as we build it out.